src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+// helper fn
+function isMan(person) {
+  return person.sex === 'm';
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredTab = people
+    .filter(isMan)
+    .filter(haveDiedInCentury);
```

Comment:
Try to avoid chaining filter() method it can cos you performance when dealing with big data. You can use one filter and combine the conditions. Also here you dont need to check the other condition if **country** is not specified.